operator/internal/controller/redpanda/redpanda_controller_test.go

```diff
@@ -247,12 +247,38 @@ func (s *RedpandaControllerSuite) minimalRP(useFlux bool) *redpandav1alpha2.Redp
 				},
 				Statefulset: &redpandav1alpha2.Statefulset{
 					Replicas: ptr.To(1), // Speed up tests ever so slightly.
+					ExtraVolumeMounts: ptr.To(`- name: test-extra-volume
```

Comment:
Rather than poluting all tests in this suite can this instead be a standalone test to exercise `tpl`? That way we'll keep the tests focused and not have to incur the additional overhead of another container on every test run.

Reply:
Also could you include an assertion in said test to verify that this template gets evaluated to what you expect?

Reply:
Sure. Will fix in the next push.